components/enterprise/Features.vue

```diff
@@ -73,9 +73,9 @@
                         <div class="card-body">
                             <div class="cart-title-container">
                                 <span class="card-icon"><Security /></span>
-                                <h4 class="card-title">Best in class support</h4>
```

Comment:
Would be nice to keep a part about Onboarding and training (instead of just support) since it's a feature sold in the EE solution

Reply:
I agree this is helpful to mention, but this section is about features, support is not a feature. I moved the support more prominently :) 

![image](https://github.com/kestra-io/kestra.io/assets/86264395/9bb9af83-bbdc-4b76-8e60-c505d7ec5fce)

Reply:
and here 
![image](https://github.com/kestra-io/kestra.io/assets/86264395/e34dab29-aef9-4f84-b8f3-d3e1189d9862)

Reply:
Nice, will need a proper block in a few months to deliver more income of the training (in a section dedicated to this part)